lib/SetAPI/sampleset/SampleSetInterface.py

```diff
@@ -21,15 +24,41 @@ def _ws_get_obj_name(self, obj_id):
         info = self.ws_client.get_object_info_new({"objects": [{'ref': obj_id}]})[0]
         return info[1]
 
+    def _dfu_get_obj_data(self, obj_ref):
```

Comment:
This could probably be moved to a generic utility module. It might be useful outside of just the SampleSetInterface.

Reply:
👍